docs/usage-with-routing.stories.mdx

```diff
@@ -74,9 +103,11 @@ An example of how to use the Carbon `Button` and `Link` components as links with
 Note the use of the `useNavigate` hook from `react-router-dom`.
 
 ```jsx
-import React, { useCallback } from "react";
+import React, { useCallback, useState } from "react";
+import { createBrowserHistory } from "history";
```

Comment:
The example below doesn't use `createBrowserHistory` anywhere, I think you can get rid of this line